Update README.md (#126)
firebase/firebase-tools-ui: README.md

# Firebase Tools UI

The web UI for Firebase Emulator Suite. Features include:

- Overview of Emulators running
- Firebase Realtime Database Data Viewer/Editor

> - More to come...

Reviewer: ```suggestion
- Cloud Firestore Data Viewer/Editor
- Logs Viewer with powerful filters
```

NOTE: The `emulators:exec` command is necessary to set the environment variables for the web app to talk to emulators.

### Developing within a user project

> To run the emulator suite in your project folder:

Reviewer: ```suggestion
You can also start the dev server of GUI and connect to your real project. To do so, first start the emulator suite in your project folder:
```

```bash
cd project/
firebase emulators:start --import my-data

✔  hub: emulator hub started at http://localhost:4400
```

> This will run the emulators for you project.<br />

Reviewer: ```suggestion
This will run the emulators for your project.<br />
```

> Run the Emulator Suite UI from the firebase-tools-ui folder:

Reviewer: ```suggestion
In another terminal, run the Emulator Suite UI from the firebase-tools-ui folder: (note that `<project-id>` must be replaced with the matching project id of your project.)
```
